IdentityAccess/src/Kreta/IdentityAccess/Infrastructure/Symfony/DependencyInjection/Compiler/OverrideUserDataTransformerServicePass.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Kreta package.
+ *
+ * (c) Beñat Espiña <benatespina@gmail.com>
+ * (c) Gorka Laucirica <gorka.lauzirika@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Kreta\IdentityAccess\Infrastructure\Symfony\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class OverrideUserDataTransformerServicePass implements CompilerPassInterface
```

Comment:
Maybe instead of using conmpiler passes its cleaner to overide using `service.yml` file. WDYT?